src/foam/nanos/dig/DUGRule.js

```diff
@@ -182,6 +214,30 @@ foam.CLASS({
     {
       name: 'createdByAgent',
       hidden: true
+    },
+    {
+      class: 'Reference',
+      of: 'foam.nanos.auth.User',
+      name: 'actingUser',
```

Comment:
do we want user to configure actingUser? or it's supposed to be the createdBy?

Reply:
Default to createdBy, but also ability to manually set

Reply:
if someone eg. intuit-admin has the permission to update the property, can they set actingUser=root?

Reply:
Yes, they would do that, another concern I have is that anyone could use postman to set the secureDaoKey to `bareUserDAO` or something similar, as the daoKey is just a string.  I giving a group the permission to edit/create rules in general is quite risky.

Reply:
@nanoscott @jlhughes what is your thought on this?

Reply:
yes .. these are meant to be configurations that only we as super-admins can do .. the users should not able to set these themselves.

Reply:
Can we do this with rule templates - so they can copy an existing and then only set certain properties?

Reply:
Added verification to the daoKey